packages/SwingSet/src/devices/bridge/device-bridge.js

```diff
@@ -12,15 +12,15 @@ function sanitize(data) {
 }
 
 /**
- * @typedef {object} BridgeDevice
- * @property {(dstID: string, obj: any) => any} callOutbound
- * @property {(handler: { inbound: (srcID: string, obj: any) => void}) => void} registerInboundHandler
+ * @typedef {object} BridgeRoot
+ * @property {(tag: string, obj: any) => any} callOutbound
+ * @property {(handler: { inbound: (tag: string, obj: any) => void }) => void} registerInboundHandler
+ * @property {() => void} unregisterInboundHandler
```

Comment:
Can we use a term that is less overloaded than "tag", ideally something that implies uniqueness within a scope? I'd like this typedef to make sense even if it takes a long time for renaming to land.
```suggestion
 * @typedef {object} BridgeRoot
 * An object representing a bridge device from which messages can be received
 * via a handler and to which messages can be sent.
 * For multiplexing, each inbound or outbound message is associated with a string identifying its channel.
 * @property {(channelName: string, obj: any) => any} callOutbound
 * @property {(handler: { inbound: (channelName: string, obj: any) => void }) => void} registerInboundHandler
 * @property {() => void} unregisterInboundHandler
```